recipe/meta.yaml

```
@@ -1,26 +1,29 @@
-{% set version = "2.0.0" %}
+{% set name = "sphinxcontrib-htmlhelp" %}
+{% set version = "2.1.0" %}
 
 package:
-  name: sphinxcontrib-htmlhelp
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/s/sphinxcontrib-htmlhelp/sphinxcontrib-htmlhelp-{{ version }}.tar.gz
-  sha256: f5f8bb2d0d629f398bf47d0d69c07bc13b65f75a81ad9e2f71a63d4b7a2f6db2
+  url: https://pypi.org/packages/source/s/sphinxcontrib-htmlhelp/sphinxcontrib_htmlhelp-{{ version }}.tar.gz
+  sha256: c9e2916ace8aad64cc13a0d233ee22317f2b9025b9cf3295249fa985cc7082e9
 
 build:
   number: 0
-  noarch: python
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
 
 requirements:
   host:
-    - python >=3.6
+    - python  >=3.9
```

```suggestion
    - python  3.9
```
Best practice in that case it to pin to the lowest python.

---

In the case of a noarch build?